src/com/yetanalytics/lrs_admin_ui/views/status.cljs

```diff
@@ -0,0 +1,227 @@
+(ns com.yetanalytics.lrs-admin-ui.views.status
+  (:require
+   [reagent.core :as r]
+   [re-frame.core :refer [subscribe dispatch]]
+   [clojure.string :as cs]
+   [com.yetanalytics.lrs-admin-ui.functions :as fns]
```

Comment:
Might want to consolidate the "functions" and your "util" namespaces

Reply:
Yeah I guess everything in there could go under like `functions/time.cljs` or something.

Reply:
🎶 `functions.time`, grab your friends... 🎶  https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/708c4bea623babe75fea085cba4d6c5a23f4ed93